website/versioned_docs/methods-wip/methods.md

```diff
@@ -0,0 +1,17 @@
+## Methods
+
+# //TODO Add some introduction why and how to use it?
+
+To use `methods` import them to your file, where you would like to use in steps.
+
+
+```javascript
+import { methods, When } from 'kakunin'
+
+When(/^I click the "([^"]*)" element$/, function(elementName) {
```

Comment:
That might be good if you inform somewhere in the `methods` section that you can use it also in e.g. form_handlers and it's just an example based on the stepDefinition.